packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.constants.js

```diff
@@ -4,7 +4,28 @@ export const INACTIVE_SUPPORT_LEVEL = ['premium-accredited'];
 
 export const AVAILABLE_SUPPORT_LEVEL = ['business', 'enterprise'];
 
-export const FULL_SUPPORT_LEVEL_SUBSIDIARIES = ['FR'];
+export const FULL_SUPPORT_LEVEL_SUBSIDIARIES = [
+  'ASIA',
+  'AU',
+  'CA',
+  'CZ',
+  'DE',
+  'ES',
+  'FI',
+  'FR',
+  'GB',
+  'IE',
+  'IT',
+  'LT',
+  'NL',
+  'PL',
+  'PT',
+  'QC',
+  'SG',
+  'SN',
+  'WE',
+  'WS',
+];
```

Comment:
Why not reverse the logic here and instead of listing the AVAILABLE SUBSIDIARIES, list the UNAVAILABLE_SUBS which will be a shorter and will ease the maintenance ?

Reply:
I thought about it... But decided not to disturb the logic that is already prodded